pages/news/index.js

```diff
@@ -1,14 +1,36 @@
 import React from 'react'
-import {Box, Text} from '@primer/components'
+import {Box, Text, Heading, FlexContainer, Relative, FlexItem} from '@primer/components'
 import Nav from '../../src/Nav'
+import whatsNew from '../../src/whats-new'
+import NewsList from '../../src/NewsList'
+import NewsImage from '../news-illo.svg'
 
-export default function News() {
+const NewsIndex = () => {
   return (
     <Box>
       <Nav />
-      <Box p={4}>
-        <Text fontSize={5}>What’s new</Text>
+      <Box className="container-xl" p={3} py={7} style={{overflow: 'hidden'}}>
+        <FlexContainer justifyContent="space-between" flexDirection={['column', 'column', 'row', 'row']}>
```

Comment:
The team pages are built to have 5 breakpoints, we might want to match that here